cli/src/cli_util.rs

```diff
@@ -1917,14 +1986,18 @@ pub enum WorkingCopyFreshness {
     WorkingCopyStale,
     /// The working copy is a sibling of the latest operation.
     SiblingOperation,
+    /// The working copy's operation could not be loaded.
+    MissingOperation,
 }
 
 impl WorkingCopyFreshness {
     /// Returns true if the working copy is not updated to the current
     /// operation.
     pub fn is_stale(&self) -> bool {
         match self {
-            WorkingCopyFreshness::Fresh | WorkingCopyFreshness::Updated(_) => false,
+            WorkingCopyFreshness::Fresh
+            | WorkingCopyFreshness::Updated(_)
+            | WorkingCopyFreshness::MissingOperation => false,
```

Comment:
It's unclear whether `MissingOperation` should be considered "stale", but returning `true` will remove `!recovered` condition from the caller.

Reply:
Actually, if we return `MissingOperation` to where the caller (`cmd_workspace_update_stale`) checks `!recovered`, that can only happen during some sort of race, since `cmd_workspace_update_stale` already checks for that situation and then recovers. Maybe I should inline `is_stale` into `cmd_workspace_update_stale` first.

Reply:
I've inlined `is_stale` as I described and also treated `MissingOperation` differently.